Makefile

```diff
@@ -351,4 +351,18 @@ test:
 	skipper make $(SKIPPER_PARAMS) _test
 
 _test: $(REPORTS)
+	if [ -d /tmp/assisted_test_infra_logs ]; then rm -rf /tmp/assisted_test_infra_logs; fi
+	mkdir /tmp/assisted_test_infra_logs
+	if [ -d /tmp/test_images ]; then rm -rf /tmp/test_images; fi
```

Comment:
add all those stuff to make clean target and delete all needed folders there.

Reply:
Fixed